src/cli/onefuzz/api.py

```diff
@@ -546,6 +546,38 @@ def get(self, vm_id: UUID_EXPANSION) -> models.Repro:
             "GET", models.Repro, data=requests.ReproGet(vm_id=vm_id_expanded)
         )
 
+    def get_files(
+        self,
+        report_container: primitives.Container,
+        report_name: str,
+        include_setup: bool = False,
+        output_dir: primitives.Directory = primitives.Directory("."),
+    ) -> None:
+        """downloads the files necessary to locally repro the crash from a given report"""
+        report_bytes = self.onefuzz.containers.files.get(report_container, report_name)
+        report = json.loads(report_bytes)
+
+        self.logger.debug(
+            "downloading files necessary to locally repro crash %s",
+            report["input_blob"]["name"],
+        )
+
+        self.onefuzz.containers.files.download(
+            report["input_blob"]["container"],
+            report["input_blob"]["name"],
+            os.path.join(output_dir, report["input_blob"]["name"]),
+        )
+
+        if include_setup:
+            setup_container = list(
+                self.onefuzz.jobs.containers.list(
+                    report["job_id"], enums.ContainerType.setup
+                )
+            ).pop()
```

Comment:
Is there an unusual reason we wrap this in a list and then pop it out?

Reply:
Nope, the goal here was just to get the lone key out of the returned dictionary. Besides using `[0]` instead of `pop()` are there any other better ways to do this?

Reply:
```py
def list(
        self,
        job_id: UUID_EXPANSION,
        container_type: enums.ContainerType,
    ) -> Dict[str, List[str]]:
        """
        List the files for all of the containers of a given container type
        for the specified job
        """
        ...
```
Despite the method name, it actually returns a dictionary.